common/util/src/testFixtures/java/org/eclipse/dataspaceconnector/common/annotations/AwsS3IntegrationTest.java

```diff
@@ -12,9 +12,8 @@
  *
  */
 
-package org.eclipse.dataspaceconnector.aws.testfixtures.annotations;
+package org.eclipse.dataspaceconnector.common.annotations;
```

Comment:
Not sure if we need to moved these existing annotations. They were kept out of common module as they were specific to a particular extension/endpoint requirement e.g. Cosmos DB , Storage etc.

Maybe its a good idea to keep them as before unless we have a need to do this change.

Reply:
Ok given your feedback I reverted the change.